NEWS.md

```diff
@@ -1,5 +1,7 @@
 # testdat (development version)
 
+* Bug fix for `expect_base()` where the test would pass when the `var` argument isn't a variable in the test data (#72, @lachlangh). 
```

Comment:
Reword a little (see https://style.tidyverse.org/news.html#bullets). The reference should be #71 (the issue) rather than #72 (the PR).

For e.g.:
``* `expect_base()` now errors when `var` doesn't exist in the test data instead of passing (#71, @lachlangh).``

Reply:
@gorcha thanks for picking up on that! All fixed